app/src/main/java/com/github/se/studybuddies/ui/groups/GroupMembers.kt

```diff
@@ -269,3 +273,96 @@ fun MemberOptionButton(
     }
   }
 }
+
+@Composable
+fun AddMemberButtonList(isBoxVisible: MutableState<Boolean>) {
+
+  Column {
+    Button(
+        onClick = { isBoxVisible.value = true },
+        shape = MaterialTheme.shapes.medium,
+        colors =
+            ButtonDefaults.buttonColors(
+                containerColor = Blue,
+            )) {
+          Text(stringResource(R.string.add_member_from_list), color = Color.White)
+        }
+  }
+}
+
+@Composable
+fun ShowContact(
+    groupUID: String,
+    groupViewModel: GroupViewModel,
+    isBoxVisible: MutableState<Boolean>
+) {
+  groupViewModel.getAllFriendsGroup(groupViewModel.getCurrentUser())
+  val members by groupViewModel.membersGroup.observeAsState()
+  members?.let {
+    Box(modifier = Modifier.fillMaxSize()) {
+      LazyColumn(modifier = Modifier.fillMaxSize()) {
+        item { Spacer(modifier = Modifier.height(64.dp)) }
+        item {
+          Box(modifier = Modifier.fillMaxSize()) {
+            IconButton(
+                onClick = { isBoxVisible.value = false },
+                modifier = Modifier.align(Alignment.TopEnd).testTag("close_contact_list")) {
+                  Icon(
+                      imageVector = Icons.Default.Close,
+                      modifier = Modifier.size(40.dp),
+                      contentDescription = "Close friends List",
```

Comment:
I don't think this icon is necessary, since it does the same thing as the go_back button. Unless it has a purpose I misunderstood, I believe you can remove it

Reply:
It is useful when there is no goBack button, which is not the case in this situation but at other place yes. And I don't think it's bad to have the cross to cancel the add friend menu even if you can use goBack.